rotkehlchen/api/rest.py

```diff
@@ -4861,33 +3970,6 @@ def _detect_evm_tokens(
             'status_code': HTTPStatus.OK,
         }
 
-    def detect_evm_tokens(
-            self,
-            async_query: bool,
-            only_cache: bool,
-            addresses: Optional[list[ChecksumEvmAddress]],
-            blockchain: SUPPORTED_EVM_CHAINS,
-    ) -> Response:
-        manager = self.rotkehlchen.chains_aggregator.get_chain_manager(blockchain)
-        if addresses is None:
-            addresses = self.rotkehlchen.chains_aggregator.accounts.get(blockchain)
```

Comment:
This is part of the logic that is missing in the method that you made async. You have to be careful with those. Also can you make sure that this is properly tested if is not?

Reply:
Why do you think that it is missing? Isn't it present here https://github.com/rotki/rotki/blob/ae7b18368fc4cbc0b45a94808d102287d6371b52/rotkehlchen/api/v1/resources.py#L2702 ?

Reply:
Didn't expect it to me moved to the resource file. Okey then

Reply:
you were right not to expect it to move there. It should not have. Refer to my comment in import data.